index.html

```diff
@@ -80,6 +80,10 @@
         <button data-action="log-data">Log segments/points</button>
       </div>
 
+      <ul id="segments">
```

Comment:
What I would do is output two tables, one for segments, the other for points. The segments table would show the `startTime`, `endTime`, `id`, and `labelText` attributes, and the points table would show `timestamp`, `id`, and `labelText`.